pyfftw/interfaces/cache.py

```diff
@@ -70,14 +70,14 @@
 
 The actual implementation of the cache is liable to change, but the
 documented API is stable.
-'''
 
+'''
 try:
     import threading as _threading
     _threading_import_error = None
 except ImportError as e:
     _threading_import_error = e
-    _threading == None
+    _threading = None
```

Comment:
should be `=` instead of `==` right?

Reply:
Yes, good catch. I guess the non-threading path never gets tested!